web/content/release-3.8.adoc

```diff
@@ -10,14 +10,23 @@ Use the top menu bar for navigation for all guides.
 
 === Quickstart
 
+* link:/{FOREMAN_VER}/Quickstart/index-foreman-el.html[Foreman on RHEL/CentOS Quickstart Guide]
+* link:/{FOREMAN_VER}/Quickstart/index-foreman-deb.html[Foreman on Debian/Ubuntu Quickstart Guide]
```

Comment:
These guides are not ready: https://docs.theforeman.org/3.8/Quickstart/index-foreman-el.html & https://docs.theforeman.org/3.8/Quickstart/index-foreman-deb.html say so.